[control-plane-manager] merge the monitoring-kubernetes-control-plane module into the control-plane-manager (#20734)
deckhouse/deckhouse: docs/documentation/pages/admin/configuration/monitoring/OVERVIEW.md

## Control plane monitoring

> Control plane monitoring is a built-in capability of the [`control-plane-manager`](/modules/control-plane-manager/) module. It organizes secure metrics collection and provides a basic set of monitoring rules for the following cluster components:

Reviewer: ```suggestion
Control plane monitoring is provided by the [`control-plane-manager`](/modules/control-plane-manager/) module. It organizes secure metrics collection and provides a basic set of monitoring rules for the following cluster components:
```
